tests/run/const_variable.pyx

```diff
@@ -0,0 +1,29 @@
+# mode: run
+
+cdef const int int_constant = 50
+cdef const int int_constant2 = int_constant
+cdef const int int_constant3
+int_constant3 = 5
```

Comment:
Does it make sense to allow setting a constant value after it's declaration?
C does not allow it and it just seems wrong to allow something like this:
```cython
cdef const int X

cdef function():
    # with a lot
    # of code
    # inside

X = 3
```

Reply:
The same goes for an uninitialized constants

Reply:
> Does it make sense to allow setting a constant value after it's declaration?

No it does not but I was not able to restrict it in the code. Currently, (if I am not wrong) it is not possible to check in the code whether is assignment part of declaration or not.